config/webpack.dev-stage.config.js

```diff
@@ -124,28 +125,25 @@ module.exports = merge(commonConfig, {
       },
       {
         test: /\.(jpe?g|png|gif)(\?v=\d+\.\d+\.\d+)?$/,
-        use: [
-          'file-loader',
-          {
-            loader: 'image-webpack-loader',
-            options: {
-              mozjpeg: {
-                progressive: true,
-                quality: 65,
-              },
-              gifsicle: {
-                interlaced: false,
-              },
-              pngquant: {
-                quality: [0.65, 0.90],
-                speed: 4,
-              },
-            },
-          },
-        ],
+        loader: 'file-loader',
       },
     ],
   },
+  optimization: {
+    minimizer: [
+      '...',
```

Comment:
What is this for?

Reply:
I believe that would be to extend default minimizer. [Package setup details](https://webpack.js.org/plugins/image-minimizer-webpack-plugin/)